content/en/opentelemetry/integrations/runtime_metrics/_index.md

```diff
@@ -9,36 +9,202 @@ type: multi-code-lang
+[6]: https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/runtime-telemetry/runtime-telemetry-java17/library
+
+{{% /tab %}}
+
+{{% tab "Go" %}}
+
+OpenTelemetry (OTel) Go applications are [instrumented manually][3]. To enable runtime metrics, see the documentation for the [runtime package][4].
+
+
+[3]: https://opentelemetry.io/docs/instrumentation/go/manual/
+[4]: https://pkg.go.dev/go.opentelemetry.io/contrib/instrumentation/runtime
+
+{{% /tab %}}
+
+{{% tab ".NET" %}}
+
+<div class="alert alert-warning">The minimum supported version of the .NET OpenTelemetry SDK is <a href="https://github.com/open-telemetry/opentelemetry-dotnet/releases/tag/core-1.5.0">1.5.0</a></div>
+
+If you have instrumented your .NET applications with [OpenTelemetry automatic instrumentation][3], runtime metrics are automatically enabled. 
+
+If you have instrumented your .NET application with [OpenTelemetry manual instrumentation][4], see the documentation for the [OpenTelemetry.Instrumentation.Runtime library][5].
+
+The default metric export interval for the .NET OTel SDK is different from the default for the Datadog .NET SDK. Datadog recommends setting the [OTEL_METRIC_EXPORT_INTERVAL][7] environment variable on your .NET service to match the default Datadog metric export interval for viewing integration metric graphs:
+- `OTEL_METRIC_EXPORT_INTERVAL=10000`
 
-## Setup
+[3]: https://opentelemetry.io/docs/instrumentation/net/automatic/
+[4]: https://opentelemetry.io/docs/instrumentation/net/manual/
+[5]: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.Instrumentation.Runtime
+[7]: https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/#periodic-exporting-metricreader
 
-Select your language to see instructions for setting up and configuring the OpenTelemetry SDK to send runtime metrics:
+{{% /tab %}}
 
-{{< partial name="opentelemetry/otel-runtime-metrics.html" >}}
-<br/>
+{{< /tabs >}}
 
 ## View runtime metric dashboards
 
-After setup is complete, see your runtime metrics in the a service's details page (see Java example below), the flame graph metrics tab, and in [default runtime dashboards][7].
+After setup is complete, see your runtime metrics in the service's details page (see Java example below), the flame graph metrics tab, and in [default runtime dashboards][7].
 
 {{< img src="opentelemetry/otel_runtime_metrics_service_page.png" alt="Service page showing OpenTelemetry runtime metrics on the JVM Metrics tab" style="width:100%;" >}}
 
+## Data collected
+
+When using OpenTelemetry runtime metrics with Datadog, you receive both the original OpenTelemetry runtime metrics as well as mapped Datadog runtime metrics for equivalent metrics. The OpenTelemetry runtime metrics have the following prefixes based on their source:
+
+| OTel Collector Datadog Exporter | Datadog Agent OTLP Ingest |
```

Comment:
Not sure the best way to format this, but would be good to include DDOT (Datadog Agent w/ OpenTelemetry Collector) in this chart.
```suggestion
| OTel Collector Datadog Exporter | Datadog Agent OTLP Ingest / DDOT |
```

Reply:
Sure that works